aws-powertools/powertools-lambda-typescript
# build(deps): bump e2e dependencies metrics

## Description of your changes

This PR supersedes:
#341
#343
#345
#346
#357

### How to verify this change

Check that github activities run successfully, optionally checkout branch & run e2e tests.

### Related issues, RFCs

N/A

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
